index.html

```diff
@@ -265,6 +207,66 @@ <h2 >Consult A Doctor</h2>
       </div>
   </section>
 
+  <!-- About Section -->
+  <section id="about">
+    
+        <h2>About Us</h2>
+        <h3><span style="color: rgb(91, 134, 150);">MEET OUR MEMBERS</span></h3>
+      
+
+      <div class="box">
+
+        <div class="card">
+          <img src="/images/shayan.jpeg" alt="shayan" loading="lazy">
+          <h5>Shayan Pal</h5>
+          <div class="pra">
+            <p>Member</p>
+            <a href="https://bio.link/shayanpal"><button>Let's Connect</button></a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img src="/images/soumyadeep.jpeg" alt="soumyadeep"loading="lazy">

```

Comment:
There should be a space between each attribute.
For writing clean and nice, it is better to give a space between them.
```diff
- alt="soumyadeep"loading="lazy"
+ alt="soumyadeep" loading="lazy"
```